src/IBM.Watson.TextToSpeech.v1/Test/Integration/TextToSpeechServiceIntegrationTests.cs

```diff
@@ -87,6 +89,74 @@ public void Synthesize_Success()
 
             Assert.IsNotNull(synthesizeResult.Result);
         }
+
+        [TestMethod]
+        public void Synthesize_sockets_SuccessAsync()
+        {
+            service.WithHeader("X-Watson-Test", "1");
+
+            // path to file
+            string Name = @"FILEPATH/yourfiletest.mpeg";
+            FileStream fs = File.Create(Name);

```

Comment:
Is this being deleted at the end of the test?

Reply:
Oh this test should be skipped, it's just for a reference on how the function works